internal/kubernetes/helpers/helpers.go

```diff
@@ -0,0 +1,110 @@
+package helpers
+
+import (
+	"fmt"
+	kubernetes "github.com/falco-talon/falco-talon/internal/kubernetes/client"
+	"github.com/falco-talon/falco-talon/utils"
+	corev1 "k8s.io/api/core/v1"
+	"reflect"
+	"strconv"
+	"strings"
+)
+
+func GetOwnerKind(pod corev1.Pod) (string, error) {
+	if len(pod.OwnerReferences) == 0 {
+		return "", fmt.Errorf("no owner references found")
+	}
+	return pod.OwnerReferences[0].Kind, nil
+}
+
+func GetOwnerName(pod corev1.Pod) (string, error) {
+	if len(pod.OwnerReferences) == 0 {
+		return "", fmt.Errorf("no owner references found")
+	}
+	return pod.OwnerReferences[0].Name, nil
+}
+
+func VerifyIfPodWillBeIgnored(parameters map[string]interface{}, client *kubernetes.Client, pod corev1.Pod, objects map[string]string) (utils.LogLine, error, bool) {
```

Comment:
For me, the function should return (bool, string, error) and not a log line. No need to pass an `objects` struct then.
It should be a method too, and in the internal/kubernetes package.

Reply:
I did it because the function VerifyIfPodWillBeIgnored is also handling the creation of the logic structure, otherwise I'd need to GetOwnerKind inside my drain logic and deal with the proper logging based on the given type.

Also, I'm unsure how to transform it into a method, would it be a method for a **client *kubernetes.Client** receiver?

Reply:
yes a method for `kubernetes.Client`.

I guess we could create a method `func (client *kubernetes.Client) GetPodOwnerKind() string` that will return None, Deployment, Daemonset, Statefulset. 

wdyt?

Reply:
I reimplement it as a method. 

Didn't added the GetPodOwnerKind because for it to make sense as a method for client ***kubernetes.Client** receiver we'd need to call again the **kube-api-server** (to use the client itself, otherwise we'd need to configure a empty receiver), instead I kept the getPodOwner function that receives the pod and simply extracts it.

WDYT about the new implementation?